core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -335,4 +335,29 @@ package object config {
         "spark.")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val SHUFFLE_FILE_BUFFER_SIZE =
+    ConfigBuilder("spark.shuffle.file.buffer")
+      .doc("Used to set the buffer size of outputstream in shuffle writer.")
```

Comment:
can we copy the document from http://spark.apache.org/docs/latest/configuration.html ?